packages/tokens-studio-for-figma/src/app/store/remoteTokens.tsx

```diff
@@ -325,34 +326,38 @@ export default function useRemoteTokens() {
     async ({ context = api, overrides }: { context?: StorageTypeCredentials, overrides?: PushOverrides } = {}) => {
       const isFolder = 'filePath' in context && !context.filePath?.endsWith('.json');
       track('pushTokens', { provider: context.provider, isFolder });
+      let pushResult;
       switch (context.provider) {
         case StorageProviderType.GITHUB: {
-          await pushTokensToGitHub(context, overrides);
+          pushResult = await pushTokensToGitHub(context, overrides);
           break;
         }
         case StorageProviderType.GITLAB: {
-          await pushTokensToGitLab(context);
+          pushResult = await pushTokensToGitLab(context);
           break;
         }
         case StorageProviderType.BITBUCKET: {
-          await pushTokensToBitbucket(context);
+          pushResult = await pushTokensToBitbucket(context);
           break;
         }
         case StorageProviderType.ADO: {
-          await pushTokensToADO(context);
+          pushResult = await pushTokensToADO(context);
           break;
         }
         case StorageProviderType.SUPERNOVA: {
-          await pushTokensToSupernova(context);
+          pushResult = await pushTokensToSupernova(context);
           break;
         }
         case StorageProviderType.TOKENS_STUDIO: {
-          await pushTokensToTokensStudio(context);
+          pushResult = await pushTokensToTokensStudio(context);
           break;
         }
         default:
           throw new Error('Not implemented');
       }
+      if (pushResult.status && String(pushResult.status).includes('failure')) {
```

Comment:
Why does the status need to be cast to a `string`? Is it not always `status: 'failure'` or `status: 'success'`? Then it should work.

```suggestion
      if (pushResult.status && pushResult.status === 'failure')) {
```

Reply:
Updated.